README.md

```diff
@@ -13,6 +13,65 @@ State in July and August of 2017.
 
 * *To be written:* Future Work
 
+Docker Quick Start
+==================
```

Comment:
Huh, I didn't realize this shape of heading levels was a thing; I guess I blocked out [the long way](https://daringfireball.net/projects/markdown/syntax#header) in favor of the `atx-style` leading `#`.

Reply:
I don't like the long way because when you get below level 2 you have to switch to `#`, but when in Rome...